Doughnut/Player/Player.swift

```diff
@@ -53,6 +51,18 @@ final class Player: NSObject {
 
   private(set) var pausedAt: TimeInterval? = nil
 
+  var playedThreshold: Double {
+    get {
```

Comment:
Swift code generally omits the `get` keyword for readonly property getters.

SwiftLint has [a default rule](https://realm.github.io/SwiftLint/implicit_getter.html) for this and actually I've planned to enable that in the next version.